tests/flow/test_effect_schema_desync.py

```
@@ -0,0 +1,375 @@
+        # master only, prior to PR #1815. The "/0" raises ArithmeticError
+        # AFTER the CREATE clause has registered the relation type.
+        # ----------------------------------------------------------------
+        try:
+            master_g.query(
+                "CREATE ()-[:R_BAD]->() WITH 1 AS x RETURN x / 0"
+            )
+            # the projection must raise; if we get here the test premise
+            # is broken (the query accidentally succeeded)
+            self.env.assertTrue(False)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)